react-native-in-app-payments-quickstart/app/screens/HomeScreen.tsx

```diff
@@ -147,57 +120,56 @@ export default class HomeScreen extends Component {
         console.log(ex);
       }
     }
-
-    this.setState({
-      canUseDigitalWallet: digitalWalletEnabled,
-    });
+    setcanUseDigitalWallet(digitalWalletEnabled)
   }
 
-  async onApplePayRequestNonceSuccess(cardDetails) {
-    if (this.chargeServerHostIsSet()) {
+
+  const onApplePayRequestNonceSuccess = async (cardDetails: any) => {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.